Description: Modify the configuration so tht it can "just work"
 The configuration provided by upstream uses the volkszaehler middleware that
 is not packaged. Since it is easier to start from a working configuration the
 modified configurtion logs to influxdb instead.
Author: Joachim Zobel <jz-2017@heute-morgen.de> 
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/etc/vzlogger.conf
+++ b/etc/vzlogger.conf
@@ -29,9 +29,11 @@
 
     // realtime notification settings
     "push": [
+        /*
         {
             "url": "http://127.0.0.1:5582"  // notification destination, e.g. frontend push-server
         }
+        */
     ],
 
     // mqtt client support (if ENABLE_MQTT set at cmake generation)
@@ -154,6 +156,28 @@
             }
         },
         {
+            /* A minimal example that "just works" on Debian.
+               The steps needed are:
+               1. Install influxdb: sudo apt install influxdb influxdb-client
+               2. Start the client by calling influx on the command line and
+               create db: CREATE DATABASE vzlogger
+               3. Set enabled to true in the configuration below
+             */
+            "enabled": false,               // disabled meters will be ignored
+            "allowskip": false,                  // errors when opening meter may be ignored if enabled
+
+            "protocol": "random",
+            "interval": 2,
+            "max": 40.0,                    // has to be double!
+            "min": -5.0,                    // has to be double!
+            "channel": {
+                "uuid": "25a53b28-75ec-4764-a8a6-68587722ddab",
+                // For details on the influxdb configuration see vzlogger.conf.InfluxDB
+                "api": "influxdb",
+                "host": "localhost:8086"
+            }
+        },
+        {
             "enabled": false,               // disabled meters will be ignored
             "allowskip": false,                  // errors when opening meter may be ignored if enabled
 
